sdk/resourcemanager/azure-resourcemanager-eventhubs/pom.xml

```diff
@@ -46,6 +46,7 @@
 
       --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
What has broken?

Reply:
The classes `com.azure.resourcemanager.eventhubs.models.Identity` and `com.azure.resourcemanager.eventhubs.models.UserAssignedIdentity` have been changed to final class.
The detail infomation following as https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/3275631/logs/591